lib/Doctrine/Persistence/Mapping/AbstractClassMetadataFactory.php

```diff
@@ -183,7 +174,8 @@ public function getMetadataFor($className)
                     foreach ($this->loadMetadata($realClassName) as $loadedClassName) {
                         $this->cacheDriver->save(
                             $loadedClassName . $this->cacheSalt,
-                            $this->loadedMetadata[$loadedClassName]
+                            $this->loadedMetadata[$loadedClassName],
+                            0
```

Comment:
Why are we explicitly passing a lifetime now?

Reply:
Hmm, I can't remember now. I thought null was being passed before and phpstan was complaining about it, but now I can't find the null. I will undo it.

Reply:
The null was removed in a bug fix in 1.1.1

Reply:
I knew I wasn't crazy :)

Reply:
I mean....I am crazy, but not in that specific instance.